### DISCUSSION TITLE
I have the same problem on this (https://github.com/mriscoc/Ender3V2S1/issues/1126).

### DISCUSSION THREAD

**Discussion by decent001** [mriscoc/Ender3V2S1#1316] [Q&A]
I have the same problem on this (https://github.com/mriscoc/Ender3V2S1/issues/1126), but I don't know how to Flash or edit a Compile bin file.  I only using a preset Ender3V2-422-MM-MPC-20240125,  What should I do? I'm looking on youtube on how to edit mrisoc but all I found is marlin.  Can somebody help me?

**Comment by mriscoc** [comment]
Check: https://github.com/mriscoc/Ender3V2S1/issues/1126#issuecomment-1788399502

**Reply by decent001** [reply]
> Check: [#1126 (comment)](https://github.com/mriscoc/Ender3V2S1/issues/1126#issuecomment-1788399502)

How can I enable the code?  Does removing the double slash after "#define" makes the code enable?  Im new to editing codes and have no any idea.

**Comment by decent001** [comment]
Thanks for the reply, but I don't know how to  "compile with
SMOOTH_ENCODER_MENUITEMS"?
I always use the default compile files when I upgrade my Printer, Iam noob
in programming. Can you help me out?
